client/src/screens/Client/Home/components/modal/NewOrEditEvent.js

```diff
@@ -134,6 +140,9 @@ const NewOrEditEvent = ({
     patient: "",
     error: ""
   });
+  const user = JSON.parse(localStorage.getItem("user"));
```

Comment:
Wrong approach!, You should  get user from local storage only one place. Like we did in Auth Provider. Don't call again here. Use it as props.